src/components/navbar/navbar.less

```diff
@@ -0,0 +1,32 @@
+.ant-row {
+  background-color: white;
+  padding: 1em 0;
+}
+
+.ant-btn-link {
```

Comment:
I think we're going to want to avoid overwriting ant styles directly based on what I've heard about AntD. 
I've seen a `styled.Button({color: black, margin-left: 7em` syntax before but can't find any docs to link to :( 
@rymaju what's the latest on this? @bensenescu hi :) can you link to docs about that syntax?

Reply:
Still going to wait on this CSS fix before merge

Reply:
Yeah I'm a bit unsure on how to go about styling to our standards. I _think_ I may have found what @jackblanc was talking about, and I can try to make that adjustment and push a commit today. But it would be great to get an online resource just to make sure I'm doing what we're trying to do